cmd/lakectl/cmd/local.go

```diff
@@ -59,7 +60,7 @@ func getLocalArgs(args []string, requireRemote bool, considerGitRoot bool) (remo
 	}
 
 	if len(args) > idx {
-		localPath = Must(filepath.Abs(args[idx]))
+		localPath = Must(filepath.Abs(Must(homedir.Expand(args[idx]))))
```

Comment:
Suggest to split the two calls into two line so it will be easy when Must panic to undestand from the line number which one broke. (can be done by looking at the stack too)

Reply:
Will do